.agents/triage/skills/run.md

```
@@ -0,0 +1,33 @@
+
+2. **Assess**: execute `skills/assess.md` (processes all pending issues in the ledger).
+   - HITL checkpoint on first issue (see assess.md).
+   - Continue through all issues, labeling and commenting as you go.
+
+3. **Report**: execute `skills/report.md`.
+   - Generates `.artifacts/triage/report.md` and prints summary.
+
+## On Completion
+
+Print:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)